data-scripts/_generators/TxtGenerator.ts

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios'
```

Comment:
There are a lot of linting issues that are resolved if `yarn run lint` is run.
Just let it run locally and push the changes.